firefoxThemes/waifus/aqua/frame.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Ooo, what are these for?

Reply:
This is an XML declaration, similar to <!DOCTYPE html> to indicate the type of XML information being used. This line of code is optional and will be removed.

Reply:
Sorry for not being clear, what is `frame.svg` used for?

Reply:
`frame.svg` is the image added to the header area of browser. Its basically an image placed behind the tabs section which is used in every theme:

https://github.com/doki-theme/doki-theme-chrome/commit/6cee46fa658a36fbe71e69c1358d624369c702b3#diff-64fbc8401aeda61fb07064dadce6a53cb80e7ff5bd48118bda3c9858f0c6770b